content/java/concepts/calendar/terms/getCalendarType/getCalendarType.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: 'getCalendarType()' 
+Description: 'Retrieves the type or format of the calendar represented by the Calendar object, returns a string representing the calendar type' 
+Subjects: 
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags: 
+  - 'Methods'
+  - 'Classes'
+  - 'Attributes'
+CatalogContent: 
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **get CalendarType** method is a function available in the Calendar class in Java. It is used to retrieve the type or format of the calendar represented by the Calendar object. This method returns a string representing the calendar type, such as "gregory", "japanese", "buddhist", etc. This method does not accept any argument as a parameter.
+
+## Syntax
+
+```java
+public String getCalendarType()
+```
+
+## Example
+
+No parameter is passed to this method.
```

Comment:
There already is a mention of `.getCalenderType()` not requiring any parameter. So, please remove this sentence.

Reply:
@ishg-153 Removed the redundant sentence.